yarn-project/noir-protocol-circuits/src/crates/public-kernel-lib/src/hash.nr

```diff
@@ -4,14 +4,17 @@ use dep::types::{
 };
 
 pub fn compute_public_data_tree_index(contract_address: AztecAddress, storage_slot: Field) -> Field {
-    dep::std::hash::pedersen_hash_with_separator([
+    dep::std::hash::pedersen_hash_with_separator(
+        [
         contract_address.to_field(),
         storage_slot
-    ], GENERATOR_INDEX__PUBLIC_LEAF_INDEX)
+    ],
```

Comment:
As earlier. Looks like it is arrays that is messing with the formatting. @kevaundray might be of your interest as well.